resources/views/contests/show.blade.php

```diff
@@ -27,6 +27,9 @@
                     <li>
                         <a href="{{ route('split', $contest->waitingRoom->id) }}" class="button -secondary">Split</a>
                     </li>
+                    <li>
+                        <a href="{{ route('contests.user.add', $contest->id) }}" class="button -secondary">+ User</a>
```

Comment:
We can fix visuals. For example, moving the export to it's own section like I've down in a few places now.. just didn't get to all of them.